akka-bench-jmh/src/main/scala/akka/http/HttpBenchmark.scala

```diff
@@ -56,7 +56,7 @@ class HttpBenchmark {
   def shutdown() = {
     Await.ready(Http().shutdownAllConnectionPools(), 1.second)
     binding.unbind()
-    system.shutdown()
+    system.terminate()
```

Comment:
Yes, changed that everywhere (apart from one place in akka-actor-tests that we can only remove once the deprecated method goes away).